articles/quickstart/native/ios-objc/_includes/_login_centralized.md

```diff
@@ -1,14 +1,23 @@
-The first step in adding authentication to your iOS application is to provide a way for your users to log in. The fastest, most secure, and most feature-rich way to do this with Auth0 is to use the [login page](/hosted-pages/login).
+The [Auth0 hosted login page](/hosted-pages/login) is the easiest way to set up authentication in your application. We recommend using the Auth0 hosted login page for the best experience, best security and the fullest array of features.
```

Comment:
Don't use best twice

Reply:
This section has been worded this way and approved for other quickstarts: https://auth0.com/docs/quickstart/spa/angular2#add-authentication-with-auth0

Reply:
Doesn't mean it is correct. However, for this sake of consistency it's fine.